hi-ml-histopathology/src/histopathology/utils/naming.py

```diff
@@ -19,6 +20,10 @@ class SlideKey(str, Enum):
     FOREGROUND_THRESHOLD = 'foreground_threshold'
     METADATA = 'metadata'
     LOCATION = 'location'
+    PATCH_SIZE = WSIPatchKeys.SIZE.value  # 'patch_size'
```

Comment:
I would suggest renaming this to Tile_size/ tile_location (also in the above) just to keep it consistent with the TilesPipeline. I'm afraid we'll start mixing naming especially if at some point we use vision transformers that might use a different notion of patches.

Reply:
+1 on using tiles. Also, "patch" and "batch" are just very close when spoken